packages/react/src/components/accordion/accordion.stories.ts

```diff
@@ -1,30 +1,57 @@
-import { storiesOfAccordion } from "dso-toolkit";
-import { storiesOf } from "@storybook/react";
+import type { Meta } from "@storybook/web-components";
+import { AccordionArgs, accordionMeta, accordionStories } from "dso-toolkit";
+
+import readme from "@dso-toolkit/core/src/components/accordion/readme.md?raw";
+import componentsReadme from "@dso-toolkit/core/src/components/accordion/components/readme.md?raw";
+
 import { templateContainer } from "../../templates";
 import { addonsSections, alignmentSections, anchorSections, basicSections, subSections } from "./accordion.content";
 
-import readme from "./readme.md?raw";
+const meta: Meta<AccordionArgs> = {
+  ...accordionMeta({ readme: `${readme}\n${componentsReadme}` }),
```

Comment:
Leuk bedacht.

Maar ik zou dan toch nog steeds de React readme bovenaan willen zien staan.

Voorheen werd `import readme from "./readme.md?raw"` doorgegeven aan de Accordion stories factory. Nu worden de readme's van `@dso-toolkit/core` en `dso-toolkit` meegegeven.

Ik weet dat er niets in de React readme staat, maar alle implementatie readme's hebben wél de framework specifieke notatie bovenin staan. In dit geval betekent dat `<DsoAccordion>`, terwijl met jouw suggestie er `<dso-accordion>` staat.

Dus als we ze alle 3 importeren, op volgorde React, Core, HTML/CSS, dan hebben we alle informatie bij elkaar en staat toch de juiste aanroep bovenin.

Reply:
Als ik dat doe loop ik tegen het issue aan dat naast de React framework specifieke notatie er tevens de 2x <dso-accordion> staat. Ik draai mijn contraptie terug en laat alleen de React framework specifieke notatie zien, zoals het al was.


Valt me trowuens op dat op https://storybook.dso-toolkit.nl/ onder angular en React überhaupt geen Docs getoond worden ...

Reply:
Ok, dan zoeken we het later wel uit :+1: